app/views/layouts/application.html.haml

```diff
@@ -6,6 +6,7 @@
     %meta{'http-equiv' => 'X-UA-Compatible', :content => 'IE=edge'}/
 
     %link{:rel => "apple-touch-icon", :sizes => "180x180", :href => "/apple-touch-icon.png"}/
+    %link{rel: "icon", sizes: "any", href: "/icon.svg", type: "image/svg+xml"}/
```

Comment:
Chrome doesn't seem to support SVG favicons

* https://bugs.chromium.org/p/chromium/issues/detail?id=294179

At least it isn't supported in my environment

```
Google Chrome	57.0.2987.133 (Official Build) （64 ビット）
変更履歴	ec33cd0c06881d919ac0de419d829ad914e0be8f-refs/branch-heads/2987@{#887}
OS	Linux 
```